js/components/sections/projectsSection.js

```diff
@@ -17,14 +19,52 @@ const projectsSection = (selector) => {
       </div>
       <div class="row">
         <div class="col-12 col-md-6 ml-md-3 col-xl-4 ml-xl-4">
-          <a class="sub-header" href="#">ALL</a>
-          <a class="sub-header" href="#">VECTOR</a>
-          <a class="sub-header" href="#">RASTER</a>
-          <a class="sub-header" href="#">UI/UX</a>
-          <a class="sub-header" href="#">PRINTING</a>
+          <a class="sub-header filter active-filter" href="#projects">ALL</a>
+          <a class="sub-header filter" href="#projects">VECTOR</a>
+          <a class="sub-header filter" href="#projects">RASTER</a>
+          <a class="sub-header filter" href="#projects">UI/UX</a>
+          <a class="sub-header filter" href="#projects">PRINTING</a>
         </div>
       </div>
       <div id="featuredProjects_block" class="row">${featuredProjectsList('#featuredProjects_block', featuredProjectsData, imgPath)}</div> `
+
+    const elementsList =  document.getElementsByClassName("filter");
+    for (let i = 0; i < elementsList.length; i++) {
+    elementsList[i].addEventListener("click", function() {
+      const dabartinis = document.getElementsByClassName("active-filter");
+      dabartinis[0].className = dabartinis[0].className.replace("active-filter", "")
+      this.className += " active-filter";
+
+    const aktyvus = document.querySelector(".active-filter");
+      if (aktyvus.innerHTML.includes('VECTOR')) {
+        projectsFilter('vector');
+      } else if (aktyvus.innerHTML.includes('RASTER')) {
+        projectsFilter('raster');
+      } else if (aktyvus.innerHTML.includes('UI/UX')) {
+        projectsFilter('Agency');
+      } else if (aktyvus.innerHTML.includes('PRINTING')) {
+        projectsFilter('Portal');
+      } else {
+        const featurai = document.getElementsByClassName('filter-card');
+        for (let a = 0; a < featurai.length; a++) {
+          let featuras = featurai[a];
+          featuras.classList.remove('no-display');
+        }
+      }
+      function projectsFilter(card) {
+        const featurai = document.getElementsByClassName('filter-card');
+        for (let a = 0; a < featurai.length; a++) {
+          if (featurai[a].children[2].innerText !== card) {
+            let featuras = featurai[a];
+            featuras.classList.add('no-display');
```

Comment:
jei pridėti animacija cia reiktu eilute pakeisti į:
animationAdd(featuras);
            setTimeout(function () {
              featuras.classList.add('no-display');
            }, 500);

Reply:
nu zinok sudejau tas animacijas ir man nepatinka, be ju buvo graziau. skraido, sokineja, studosi elementai, belekaip. kai ner animacijos jie iskart i vietas sustoja ir kazkaip tvarkingiau ziurisi. as jas dabar uzkomentavau, galim taip ir palikt, o esant norui ir poreikiui bus galima atkomentuot ir pasileist

Reply:
👍